<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer</class>
 <widget class="QWidget" name="customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Customer Management</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rowCount">
      <number>28</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>身份证</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>手机号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>住址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联系人姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联系人电话</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联系人邮件</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联系人关系</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>客户表</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="add">
      <attribute name="title">
       <string>增加</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="17" column="0" colspan="2">
        <widget class="QPushButton" name="customerAdd">
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="CusAddRela"/>
       </item>
       <item row="16" column="0" colspan="2">
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="CusAddID"/>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="CusAddName"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>住址</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>联系人关系</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QLineEdit" name="CusAddCphone"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>身份证</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>联系人邮件地址</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>手机号</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>联系人电话</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="CusAddPhone">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLineEdit" name="CusAddAddress"/>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QLineEdit" name="CusAddCaddr"/>
       </item>
       <item row="9" column="0">
        <widget class="QLineEdit" name="CusAddCname"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>联系人姓名</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="query">
      <attribute name="title">
       <string>查询</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>身份证</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="CusID_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>手机号</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="CusName_2"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="CusPhone_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>住址</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLineEdit" name="CusAddress_2"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>联系人姓名</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>联系人关系</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="CusCname_2"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="CusRela_2"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>联系人电话</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QLineEdit" name="CusCphone_2"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>联系人邮件地址</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QLineEdit" name="CusCaddr_2"/>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QPushButton" name="clearButton_2">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QPushButton" name="CusQueryButton">
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dele">
      <attribute name="title">
       <string>删除</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="change">
      <attribute name="title">
       <string>更改</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>CusAddCaddr</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>CusAddCphone</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>CusAddCname</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>CusAddRela</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>CusAddAddress</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>CusAddName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>CusAddPhone</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>CusAddID</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
